src/test/java/com/microsoft/util/ElementUtilTest.java

```diff
@@ -55,13 +55,23 @@ public void extractPackageElements() {
     public void extractSortedElements() {

```

Comment:
For test naming, recommend 'MethodName_StateUnderTest_ExpectedBehavior'
See: https://dzone.com/articles/7-popular-unit-test-naming

Reply:
If you have multiple expected behavior here, try to separate them in different test cases.

Reply:
Thanks for the suggestion, I will create an work item to refine the test naming of this project.

Reply:
The test is target to check we get expected sorted result set after applying filters. But I think precheck needs to make the items to be exclude really exists. 

Just in case the items to exclude no longer exit, test passed but not the logic not verified.